src/playback-watcher.js

```diff
@@ -115,6 +121,31 @@ export default class PlaybackWatcher {
   }
 
   /**
+   * Fixes situations where there's a bad seek
+   *
+   * @return {Boolean} whether an action was taken to fix the seek
+   * @private
+   */
+  fixBadSeeks_() {
+    let seekable = this.seekable();
+    let currentTime = this.tech_.currentTime();
+
+    if (this.tech_.seeking() &&
```

Comment:
It's probably safer to leave this check in here, just thought I'd note that I don't think we can get into this function with seeking is false since the check `this.tech_.seeking() && this.fixBadSeeks_()` won't call `fixBadSeeks_` if seeking returns false